src/hooks/earn/useFormatDisplayEarnOpportunityData.tsx

```diff
@@ -170,7 +170,7 @@ const buildProtocolItem = (
     key: 'protocol',
     dataTestId: `protocol-${protocolValue}`,
     label: t('labels.protocol'),
-    value: protocolValue,
+    value: capitalizeString(protocolValue),
     tooltip: t('tooltips.protocol'),
     valuePrepend: (
       <EntityChainStack
```

Comment:
@mmioana I imagine `capitalizeString` will break as soon as you need uppercase anything that is not just the first letter:

Take `Upshift Finance` which is a protocol we'll release with on day 1: https://develop.jumper.exchange/earn?forYou=false&protocols=upshift+finance

also it's not rare to find names like `FUSE` or `Polygon zkEVM` on the widget.

Reply:
for what it's worth, a bit related: https://lifi.atlassian.net/browse/LF-16897?search_id=2240a590-9ea0-45d0-8a63-e91d82c0db51

I think protocols won't be straightforward, but is there something in the SDK we could use to go from chain id to chain name?